cmsplugin_articles_ai/admin.py

```diff
@@ -18,7 +18,7 @@ class ArticleAttachmentInline(admin.StackedInline):
     fields = ["name", "attachment_file"]
 
 
-class ArticleAdmin(PlaceholderAdminMixin, admin.ModelAdmin):
+class ArticleAdmin(PublisherAdmin):
```

Comment:
I have no idea why there was PlaceholderAdminMixin in the article admin because there is no PlaceholderField used in the article model.